invenio_app_ils/ext.py

```diff
@@ -9,15 +9,24 @@
 
 from __future__ import absolute_import, print_function
 
-from flask import Blueprint
+import logging
+
+from flask import Blueprint, current_app
 from invenio_indexer.api import RecordIndexer
+from invenio_rest.errors import RESTException
 from werkzeug.utils import cached_property
 
 from . import config
 from .circulation.receivers import register_circulation_signals
 from .pidstore.pids import DOCUMENT_PID_TYPE, ITEM_PID_TYPE
 
 
+def handle_rest_exceptions(exception):
```

Comment:
This should probably just go to Invenio-REST [``ext.py``](https://github.com/inveniosoftware/invenio-rest/blob/master/invenio_rest/ext.py#L79) as:

```python
app.errorhandler(RESTException)(create_api_errorhandler())
```